core/services/keystore/csa.go

```diff
@@ -90,8 +90,8 @@ func (ks *CSA) Unlock(password string) error {
 		return err
 	}
 
-	for _, key := range keys {
-		err := ks.unlockAndAddKey(&key, password)
+	for i := range keys {
+		err := ks.unlockAndAddKey(&keys[i], password)
```

Comment:
Fixes gosec G601: Implicit memory aliasing in for loop.

See https://husni.dev/beware-of-implicit-memory-aliasing-in-go-foor-loop/ for an example